artifacts/scripting/hooks.yml

```diff
@@ -0,0 +1,560 @@
+# Format:
+# - name: Human readable hook name    # HOOK_BLABLA and hs_blabla.int are generated from this
```

Comment:
I'd suggest to separate human readable `name` (or title) and hook `id` (HOOK_BLABLA constant etc.). Seeing names in PascalCase - gives a false impression that you are supposed to use the name in this form somehow.

PS: I believe these docs should be human readable in source YAML format as well as the target generated format.

Reply:
I don't understand this. It's like saying that LaTex source of a document should be human readable. The very point of generating the target is that you don't have to read the source.

Reply:
Well if we are using yaml as the source format anyway (which is very human-friendly), it's handy to be able to just open the source and read it without having to build the docs.

Reply:
Hook ids added.